ec2/README.md

```diff
@@ -39,35 +39,32 @@ These playbooks will:
     * AWS_SSH_PRIV_KEY_PATH  - Path to your private ssh key to use for the ec2 instances
 
 ### Execute
-  * `cd ec2`
+  * Navigate to the `ec2` folder
+        $ cd catasb/ec2
   * Edit the variables file `ec2/common_vars`
-    * Update:
+    * Note the following and update:
       * AWS_SSH_KEY_NAME="splice"
       * TARGET_DNS_ZONE="ec2.dog8code.com"
         * Needs to match a hosted zone entry in your Route53 account, we will create a subdomain under it for the ec2 instance
-  * `./run_create_infrastructure.sh`
-    * Creates our infrastructure in ec2 if it doesn't exist
-  * `./run_setup_environment.sh`
-    * Sets up OpenShift
-  * In Web Browser
+  * Create our infrastructure in ec2 if it doesn't exist
+        $ ./run_create_infrastructure.sh
+  * Run the setup script
+        $ ./run_setup_environment.sh
+  * Open a Web Browser
```

Comment:
You're adding a newline to try to make it easier to read, but it doesn't work. It just ends up being on the same line. I would make all the commands escaped with triple tick marks ```